_data/roadmap.yml

```diff
@@ -1,4 +1,53 @@
 future:
+  - title: Open Data Hub 1.3 - (May/June 2022)
+    version: 1.3
+    children:
+      - title: Tutorials and website cleanup
+      - title: Dashboard updates
+        description: Admin - BYON + Cluster Settings
+      - title: Bring your own notebooks (with and without Pipelines)
+        description: Allow admins the flexibility to provide to users their own notebook images
+      - title: Model Serving
+        description: Implementation of Model Mesh using Oauth
+      - title: Community governance model
+        url: https://github.com/opendatahub-io/opendatahub-community/issues/11
+      - title: Kubeflow 1.5 support
+        description: Includes ability to install Kubeflow v1.5.0 components on OpenShift
+        description: Support for the integration of Service Mesh with Kubeflow v1.5 on OpenShift
```

Comment:
Multiple descriptions won't be rendered.  We can maybe add a list here or use < /br>